taichi/ir/ir.cpp

```diff
@@ -665,7 +665,7 @@ void Block::insert(std::unique_ptr<Stmt> &&stmt, int location) {
 
 void Block::insert(VecStatement &&stmt, int location) {
   if (location == -1) {
-    location = (int)statements.size() - 1;
+    location = (int)statements.size();
```

Comment:
(I wonder why this has been in the codebase for 5 months...)

Reply:
Thanks for fixing this! It's surprising that we have this issue for so long time without noticing...